src/bindings/html/langs.js

```diff
@@ -1,4 +1,4 @@
-import { prioritizeLocales } from '../../lib/intl';
+import { prioritizeLocales } from '../../lib/shims';
```

Comment:
Not sure if we'll ever standardize `prioritizeLocales`, thus I kept it as a 'l20n intl', now a shim.